src/graphql/trusted_domain.rs

```diff
@@ -86,20 +86,31 @@ impl TrustedDomainMutation {
         Ok(name)
     }
 
-    /// Removes a trusted domain, returning the old value if it existed.
+    /// Removes multiple trusted domains, returning the old values if they existed.
     #[graphql(
         guard = "RoleGuard::new(Role::SystemAdministrator).or(RoleGuard::new(Role::SecurityAdministrator))"
     )]
-    async fn remove_trusted_domain(&self, ctx: &Context<'_>, name: String) -> Result<String> {
+    async fn remove_trusted_domains(
+        &self,
+        ctx: &Context<'_>,
+        names: Vec<String>,
+    ) -> Result<Vec<String>> {
+        let mut removed = Vec::new();
         {
             let store = crate::graphql::get_store(ctx).await?;
             let map = store.trusted_domain_map();
-            map.remove(&name)?;
+
+            for name in &names {
+                if map.remove(name).is_ok() {
+                    removed.push(name.clone());
+                }
+            }
         }
```

Comment:
According to #413, the additional block is no longer needed. Could you remove it?